tools/clean.sh

```diff
@@ -42,6 +42,9 @@ make clean
 cd $SRCDIR/tc/sgx/trusted_worker_manager/enclave
 rm -rf build deps
 
+cd $SRCDIR/examples/enclave_manager
+SGX_MODE=SIM make clean
```

Comment:
SGX_MODE is set as an environmental variable and shouldn't be hard coded here. Also if the changes in clean.sh is unrelated of this PR , can you raise a separate PR for clean.sh changes. Preferred approach is to keep unrelated changes in different PR.

Reply:
This hardcoded env setting is to address the error out logic as follows 

https://github.com/hyperledger-labs/trusted-compute-framework/blob/ab50a1368d60a36626ac171b1a72a37152ec2b5a/examples/enclave_manager/Makefile#L18-L20

If missing, the `clean` target for this folder would be broken.

Forgive me for this growing PR =_= 
Separate this change out takes effort due to too many commits now. So sincerely ask for leaving it this time, please.

I would never raise PR so large. Thanks for the good advice 😄

Reply:
Separation is done as https://github.com/hyperledger-labs/trusted-compute-framework/pull/42

Reply:
Separation goes this PR https://github.com/hyperledger-labs/trusted-compute-framework/pull/42